DIRECTOR_HANDBOOK.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Foundation<br>Board of Directors Handbook
```

Comment:
extremely minor nit, but I'm seriously not a fan of dropping `<br>` in the middle of a header. Suggest just using a `:`